face/capture/src/main/java/com/simprints/face/capture/screens/livefeedback/LiveFeedbackFragmentViewModel.kt

```diff
@@ -197,8 +277,11 @@ internal class LiveFeedbackFragmentViewModel @Inject constructor(
     /**
      * Since events are saved in a blocking way in [SimpleCaptureEventReporter.addCaptureEvents],
      * to speed things up this method creates multiple async jobs and run them all in parallel.
+     *
+     * Auto-capture generates a number of sample captures that typically significantly exceeds samplesToKeep.
+     * Thus the non-qualifying samples are excluded to avoid excessive event data.
```

Comment:
Hm, I don't see this comment reflected in the function (neither here, nor in the old file)!?